bingo/bingo-core/src/core/ringo_index.cpp

```diff
@@ -67,10 +67,8 @@ void RingoIndex::prepare(Scanner& rxnfile, Output& output, std::mutex* lock_for_
     {
         // CrfSaver modifies _context->cmf_dict and
         // requires exclusive access for this
-        auto locker = lock_for_exclusive_access ?
-                      std::unique_lock<std::mutex>(*lock_for_exclusive_access) :
-                      std::unique_lock<std::mutex>();
-        
+        auto locker = lock_for_exclusive_access ? std::unique_lock<std::mutex>(*lock_for_exclusive_access) : std::unique_lock<std::mutex>();
+
```

Comment:
IMHO, code on the left is better readable. Can allow some freedom?

Reply:
That's OK, after all it's just a simple ternary operator, nothing special.